praw/objects.py

```diff
@@ -1807,8 +1809,9 @@ def __init__(self, reddit_session, subreddit=None, page=None,
             page = json_dict['page']
         info_url = reddit_session.config['wiki_page'].format(
             subreddit=six.text_type(subreddit), page=page)
+        self._params = kwargs
         super(WikiPage, self).__init__(reddit_session, json_dict, fetch,
-                                       info_url, **kwargs)
+                                       info_url)
```

Comment:
Gotcha, but [this comment](https://github.com/praw-dev/praw/pull/583#discussion_r50685405) still applies.